src/box/lua/schema.lua

```diff
@@ -336,6 +336,14 @@ end
 
 box.is_in_txn = builtin.box_txn
 
+box.txn_id = function()
+    local id = tonumber(builtin.box_txn_id())
+    if id == -1 then
```

Comment:
Personally I'd return -1 as is to avoid throwing errors, but it is debatable.

Reply:
Probably it's better to return nil value. It's a Lua, not C :)

Reply:
Yep, for sure..